lib/acfs/errors.rb

```diff
@@ -49,7 +49,8 @@ def initialize(opts = {})
       @stubs     = opts.delete :stubs
       @operation = opts.delete :operation
 
-      super opts, 'Ambiguous stubs.'
+      super opts, "Ambiguous stubs.\n"\
+        "#{stubs.map {|s| '  ' + s.opts.pretty_inspect }.join}"
```

Comment:
This indents only the lines after the first, I removed the interpolation and use `indent 2` now.